README.md

```diff
@@ -77,7 +77,7 @@ A step by step guide to setup a full node and validator on the Joystream testnet
 ### Integration tests
 
 ```bash
-tests/network-tests/run-tests.sh
+tests/integration-tests/run-full-tests.sh
```

Comment:
This script is not yet updated, I was wondering if we should move back to `network-tests` [once the merge process of the integration tests](https://github.com/Joystream/joystream/issues/3015) is done?

Reply:
Yeah I wasn't sure which one we wanted to use keep. We can certainly update this once the merge is complete.